catalog/config.js.example

```diff
@@ -1,10 +1,24 @@
 window.QUILT_CATALOG_CONFIG = {
-  "registryUrl": "<registry url here>",
   "alwaysRequiresAuth": false,
+  "analyticsBucket": "<s3 bucket here>",
   "apiGatewayEndpoint": "https://ozj6r9bglc.execute-api.us-east-1.amazonaws.com/prod",
-  "s3Proxy": "https://open-proxy.quiltdata.com",
+  "azureBaseUrl":  "<azure url here>",
+  "azureClientId": "<client id here>",
+  "binaryApiGatewayEndpoint": "<registry url here>",
+  "googleClientId": "414254601868-vlofcqbns27cd7v337k0v8kg6g11fukr.apps.googleusercontent.com",
+  "intercomAppId": "",
+  "mixpanelToken": "",
+  "mode": "MARKETING",
```

Comment:
@drernie this default doesn't seem right for a "general" case, bc it disables all the catalog features and just shows the website